### DISCUSSION TITLE
Beatmaps lose original background if you try to change it without saving

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#23663] [Q&A] [answered]
I noticed this while checking if changing a beatmap background updates the status to `local` (which seems to work perfectly), but if I exit without saving the new background, the beatmap just loses the original one (and the status is still `ranked`). Not sure if it's the correct behaviour. I think I'm combining two issues here.
I'm also scared to test this with the audio track... (but maybe it has a different outcome)

https://github.com/ppy/osu/assets/43073074/bc74868c-4b5a-479d-882d-9744fbea1ed2

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/23669
